docs/learn/provider_specific_features.md

```diff
@@ -18,7 +18,7 @@ This feature can be extremely useful when extracting structured information or u
 
 To use structured outputs with tools, use the `OpenAIToolConfig` and set `strict=True`. You can then use the tool just like you normally would:
```

Comment:
should we add a link to the `Tools` docs somewhere here? we say "like you normally would" but then provide no reference